torch/distributed/nn/jit/instantiator.py

```diff
@@ -37,11 +38,12 @@ def get_arg_return_types_from_interface(module_interface):
 
     arg_str_list = []
     arg_type_str_list = []
+    assert method_schema is not None
     for argument in method_schema.arguments:
         arg_str_list.append(argument.name)
 
         if argument.has_default_value():
-            default_value_str = " = {}".format(argument.default)
+            default_value_str = " = {}".format(argument.default)  # type: ignore[attr-defined]
```

Comment:
Can you please change it to `default_value` rather than suppressing an error here?
```suggestion
            default_value_str = " = {}".format(argument.default_value)
```